src/carousel/Carousel.js

```diff
@@ -1235,7 +1237,7 @@ export default class Carousel extends Component {
 
         return (
             <Component style={[mainDimension, slideStyle, animatedStyle]} pointerEvents={'box-none'} {...specificProps}>
```

Comment:
Animated style is passed as a prop to this generic `Component` here. If you just pass `animatedValue` to the `renderItem` as a prop you will see duplicate animations on both the container and it's contents. Not sure if that is what we want happening.